### DISCUSSION TITLE
how to change the smtp port to 465 or 587

### DISCUSSION THREAD

**Discussion by yydsapi** [emersion/maddy#713] [Q&A] [answered]
Great product！ I have a question, how to change the smtp port to 465 or 587, I find that when sending mail, it is always blocked on port 25, thank you! This is my log line:
{"domain":"gmail.com","io_op":"dial","msg_id":"6a14e213-668d026b","reason":"dial tcp [2607:f8b0:4003:c04::1a]:25: connect: connection timed out","remote_addr":"[2607:f8b0:400

**Comment by foxcpp** [comment, marked as answer]
You are not supposed to send outbound mail using 465 or 587 port. You should contact your provider and request port unblocking or forward your messages via an external host that does not have port 25 blocked e.g. one hosted by your provider.
